command-line/src/test/java/graphql/linalyzer/cli/ExecutionTest.java

```diff
@@ -0,0 +1,41 @@
+package graphql.linalyzer.cli;
+
+import graphql.linalyzer.cli.test.utils.OutputChecker;
+import org.junit.Test;
+
+import static graphql.linalyzer.cli.test.utils.OutputChecker.fileLine;
+import static graphql.linalyzer.cli.test.utils.OutputChecker.ruleLine;
+import static graphql.linalyzer.cli.test.utils.TestFileUtils.createTempFile;
+import static java.util.Collections.singletonList;
+
+public class ExecutionTest {
+
+    @Test
+    public void testSimpleConfigAndSchema() {
```

Comment:
Before adding the yaml-based integration tests I want to check the test code with some unit tests